Palladium Fantasy 1E/sheet.json

```
@@ -0,0 +1,8 @@
+{
+	"html": "palladium_fantasy_1e.html",
+	"css": "palladium_fantasy_1e.css",
+	"authors": "Kraynic",
+	"roll20userid": "1869955",
+	"preview": "palladium_fantasy_1e.png",
+	"instructions": "ReadMe.md"
```

```suggestion
	"instructions": "Sheet for Palladium Fantasy 1E. More info can be found in the [Readme](https://github.com/Roll20/roll20-character-sheets/blob/master/Palladium%20Fantasy%201E)"
```
This way the readme will be directly linked in the `instructions` section. [Example of another sheet](https://github.com/Roll20/roll20-character-sheets/blob/master/Weaverdice/sheet.json) that links the Readme in the `instructions` section.